products/android_studio.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Android Studio Plugin Development
+---
+
+## Introduction 
+The Android Studio IDE is built on top of the IntelliJ Platform.
+This relationship means that Android Studio plugins can be created using IntelliJ IDEA, Java or Kotlin, and the IntelliJ Platform.
+Android Studio plugins can also be packaged and distributed at [plugins.jetbrains.com](https://plugins.jetbrains.com), like plugins for JetBrains IntelliJ Platform-based products.
+
+Android Studio plugins extend or add functionality to the Android Studio IDE.
+They are not Android modules or apps to be run in the Android operating system such as on a smartphone or tablet.
+
+## Configuring IntelliJ Platform Projects for Android Studio Plugin Development
+An Android Studio plugin project is an example of cross-development: plugins projects are created in IntelliJ IDEA but intended to be run in a different IntelliJ Platform-based IDE.
```

Comment:
see above, I'd shorten that paragraph by just linking to the relevant "getting started with Gradle setup" links